tasks.py

```diff
@@ -150,6 +152,12 @@ def generate_packages(context):
 # ------------------------------------------------------------------------------
 # START / STOP / DEBUG
 # ------------------------------------------------------------------------------
+@task
+def export(context):
+    """Export docker compose configuration to `compose.yaml` file."""
+    docker_compose(context, "convert > compose.yaml")
```

Comment:
Should we upstream this? Can we describe when you may want to use the exported files?

Reply:
Improved docs, should I add it to cookiecutter templates and nautobot repository as well?